datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/current/root.json

```diff
@@ -2,143 +2,143 @@
  "signatures": [
   {
    "keyid": "0005ef94b05cc7826ff2ba5bad1a87d5a8101a095be6ac3dbd1233c3eb6db029",
```

Comment:
To reviewer: this is difficult to see on Github, but 3 new keys were added and the 3 v9 keys have new signatures.